src/store/metadata/stub/content/editions/a16mu_2023-11.json

```diff
@@ -0,0 +1,127 @@
+{
+  "@context": "https://staging.idpd.uk/#ns",
+  "@id": "https://staging.idpd.uk/datasets/a16mu/editions",
+  "@type": "hydra:Collection",
+  "title": "Modelled unemployment rate aged 16 years and over",
+  "editions": [
+    {
+      "@id": "https://staging.idpd.uk/datasets/cpih/editions/2022-01",
+      "@type": "dcat:Dataset",
+      "in_series": "https://staging.idpd.uk/datasets/cpih",
+      "identifier": "2022-01",
```

Comment:
so this needs to match the identifier you've used at the end of line 91 in datasets.json, i.e needs to be "2023-11"

Reply:
fixed